### ISSUE TITLE
[Bug]: lfs tracking

### ISSUE THREAD

**Issue by kevin666aa** [microsoft/autogen#2269]
### Describe the bug

Each time I pull, I see several png files being modified locally and I cannot revert the change. I believe it is due to the github large file system used.

Also, there are two .gitattributes file, one in local and one in `website` folder, I don't know if that is intentional, or the one in the `website` folder should be removed? @davorrunje 

### Steps to reproduce

The `composio-autogen.png` is not in lfs
```bash
(base) kevin@zens-MacBook-Pro autogen % git lfs ls-files | grep "composio-autogen.png"

(base) kevin@zens-MacBook-Pro autogen % git status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   test/test_files/test_image.png
        deleted:    website/.gitattributes
        modified:   website/static/img/gallery/composio-autogen.png

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        website/docs/topics/groupchat/

no changes added to commit (use "git add" and/or "git commit -a")
(base) kevin@zens-MacBook-Pro autogen % git restore website/static/img/gallery/composio-autogen.png
Encountered 1 file that should have been a pointer, but wasn't:
        website/static/img/gallery/composio-autogen.png
```

**Comment by davorrunje** [comment]
@kevin666aa they should be removed, thanx for figuring this out!

**Comment by ekzhu** [comment]
Fixed #2270
